clang/unittests/Format/FormatTest.cpp

```diff
@@ -20794,7 +20794,7 @@ TEST_F(FormatTest, CatchAlignArrayOfStructuresRightAlignment) {
   verifyFormat("int a[][] = {\n"
                "    {\n"
                "     {0, 2}, //\n"
-               " {1, 2}  //\n"
+               " {1, 2} //\n"
```

Comment:
This one is was not aligned on purpose, just the way clang-format did handled such cases.

Reply:
We don't need to change the test case with https://github.com/llvm/llvm-project/pull/68743#discussion_r1360437969.